x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/services/mistral/response/MistralErrorResponseHelperTests.java

```diff
@@ -15,7 +15,7 @@
 
 import static org.mockito.Mockito.mock;
 
-public class LlamaErrorResponseTests extends ESTestCase {
+public class MistralErrorResponseHelperTests extends ESTestCase {
```

Comment:
Did you mean to duplicate this file or rename it? Is `LlamaErrorResponseTests.java` now missing?